onnxruntime/core/providers/cuda/tensor/slice.cc

```diff
@@ -65,9 +65,9 @@ REGISTER_V11_TYPED_SLICE(float)
 
 template <typename Tind, bool dynamic>
 Status Slice<Tind, dynamic>::ComputeInternal(OpKernelContext* ctx) const {
-  const auto* input_tensor = ctx->Input<Tensor>(0);
+  auto input_tensor = ctx->Input<Tensor>(0);
```

Comment:
Use const auto*/auto& as per coding convention. Same comment for the line below.